code/modules/antagonists/heretic/heretic_living_heart.dm

```diff
@@ -139,12 +152,15 @@
 		return FALSE
 
 	var/mob/living/carbon/human/tracked_mob = human_targets[last_tracked_name]
-	if(QDELETED(tracked_mob))
+	if(QDELETED(tracked_mob) && QDELETED(blade))
```

Comment:
Ok I get it now

You should only have 1 var for what you're tracking (you shouldn't need two) 

Change `tracked_mob` -> `tracked_atom`, and it should ultimately be some human or some sickly blade